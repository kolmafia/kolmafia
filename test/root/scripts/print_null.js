const {print} = require("kolmafia");

try {
  print(null);
} catch (e) {
  print("exception: " + e.message);
}
