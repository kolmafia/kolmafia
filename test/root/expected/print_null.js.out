Script exception: Passing null to an ASH function is not supported. (file:%%STACKTRACE_LOCATION%%/print_null.js#4)
